wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -153,6 +153,24 @@ public Rotation2d rotateBy(Rotation2d other) {
         m_cos * other.m_cos - m_sin * other.m_sin, m_cos * other.m_sin + m_sin * other.m_cos);
   }
 
+  /**
+   * Returns the absolute value of the Rotation2d 
+   * 
+   * @return The new absolute value Rotation2d
+   */
+  public Rotation2d abs() {
```

Comment:
What's a use case for this?

Reply:
Here's something I have been working on, it decides when to follow forwards or backward back on which side is facing closest to the direction it wants to travel. This is just something simple because I often want to compare which angle is furthest from 0 (which is why I asked about the comparator interface)

heading = current heading
direction = desired heading
```java
            if (Math.abs(heading.minus(direction).getRadians()) < Math.PI) {
                followDirection = FollowDirection.FORWARD;
            } else {
                followDirection = FollowDirection.BACKWARD;
            }
```

Reply:
I'm skeptical this does what you want. I'd have to see unit tests for comparison though. The most straightforward way to do what you want is what you wrote in the snippet above.